lib/diagnostic.rb

```diff
@@ -105,15 +119,20 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+    p 'ROAR'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+"Don't know."
```

Comment:
Composition allows a class to draw from multiple sources whereas inheritance only allows a class to draw from one source.